applications/GeoMechanicsApplication/tests/test_mohr_coulomb_with_tension_cutoff/test_dirichlet_corner_return_zone/ProjectParameters.json

```
@@ -0,0 +1,167 @@
+{
+    "problem_data": {
+        "problem_name":         "test_dirichlet_corner_return_zone",
+        "start_time":           0.0,
+        "end_time":             1.0,
+        "echo_level":           1,
+        "parallel_type":        "OpenMP",
+        "number_of_threads":    1
+    },
+    "solver_settings": {
+        "solver_type":                        "U_Pw",
+        "model_part_name":                    "PorousDomain",
+        "domain_size":                        2,
+        "model_import_settings":              {
+            "input_type":       "mdpa",
+            "input_filename":   "mesh"
```

When moving the identical `mesh.mdpa` files to subdirectory `common`, we need to update this path as follows:
```suggestion
            "input_filename":   "../common/mesh"
```